apps/condo/domains/miniapp/queries/B2BAppNewsSharingConfig.graphql

```diff
@@ -0,0 +1,5 @@
+query getB2BAppNewsSharingConfig {
```

Comment:
why only Id?

Seems like you really just need to use count query here

And you also need to only get B2BAppNewsSharingConfigs that are active and are public

Consider this case:

I, as a big construction company create a miniapp integraion with my internal systems. This miniapp can have news sharing function. This miniapp is private and cannot be used by other managing companies

And judging by this query and by your code, u will show every user from every organization a prompt to go and check other services